java-checks/src/main/java/org/sonar/java/checks/naming/KeywordAsIdentifierCheck.java

```diff
@@ -33,7 +33,7 @@ public class KeywordAsIdentifierCheck extends BaseTreeVisitor implements JavaFil
 
   private JavaFileScannerContext context;
 
-  private static final Set<String> FORBIDDEN_IDENTIFIERS = SetUtils.immutableSetOf("enum", "_","assert");
```

Comment:
If we decide to change the implementation then the [RSPEC](https://github.com/SonarSource/rspec/blob/master/rules/S1190/java/rule.adoc) for [S1190](https://sonarsource.github.io/rspec/#/rspec/S1190) should be updated accordingly